README.md

```diff
@@ -250,6 +255,9 @@ bohb = @hyperopt for i=18, sampler=Hyperband(R=50, η=3, inner=BOHB(dims=[Hypero
 end
 ```
 
+When using BOHB, a Kernel Density Estimator will estimate hyperparameters that balance exploration and exploitation based on previous observations. It does this by setting the `state` variable to a tuple of the estimated set of hyperparameters, at certain points within the loop. As a consequence, the `state` returned (`Optim.minimizer(res)` in the previous example) needs to be a tuple holding the values for each hyperparameter in order (`(a, c)` in the previous example).
```

Comment:
Is it true that the state has to be the parameters in order? Can the state not be anything?

Reply:
The state, as returned by the KDE, is a tuple of hyperparameters. I am pretty sure that returning a state as anything else would not fit into the "previous observations" machinery. Even if you could return something else, then the `state` variable would take on two different forms, which I can not imagine will run without problem.

This is the problem I have talked about before in [#79](https://github.com/baggepinnen/Hyperopt.jl/issues/79)

Reply:
Ah, my confusion comes from BOHB as inner sampler behaving differently, thanks for clarifying!

Reply:
Right, that is a point that I also had to get used to ^_^
So...resolved?